src/dso_api/dynamic_api/views/mvt.py

```diff
@@ -34,12 +42,30 @@ def get_context_data(self, **kwargs):
 
         return context
 
-    def _get_geo_tables(self, schema):
-        """Yields names of tables that have a geometry field."""
-        for table in schema.tables:
-            for field in table.fields:
-                if field.is_geo:
-                    yield to_snake_case(table.name)
+
+class DatasetMVTSingleView(TemplateView):
+    """Shows info about a dataset and its geo-tables."""
+
+    template_name = "dso_api/dynamic_api/mvt_single.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        try:
+            ds = Dataset.objects.get(name=kwargs["dataset_name"])
+        except Dataset.DoesNotExist:
+            raise Http404("Invalid dataset") from None
```

Comment:
```suggestion
        ds = get_object_or_404(Dataset, name=kwargs["dataset_name"])
```

Kan net zo makkelijk zijn. :-)
`from django.shortcuts import get_object_or_404`